data/changelog.txt

```diff
@@ -1,5 +1,22 @@
 # 0.F (2021-06-19)
 
+## Highlights
+Nested Containers rationalize inventory management and enable dropping and retrieving go-bags during fights.
+Achievements track your deeds and misdeeds across games.
+Proficiencies better represent deeper knowledge required for various endeavors, mostly crafting.
+Bleeding added to both the player and monsters as the first step toward a more comprehensive wound and wound treatment system.
+Weariness tracking added to represent longer term physical exhaustion.
+Elevated Bridges/Navigable Rivers added, allowing better navigabilty while using boats.
```

Comment:
```suggestion
Elevated bridges/navigable rivers added, allowing better navigabilty while using boats.
```

Reply:
Combination of this and actual-nh's suggestions: https://github.com/CleverRaven/Cataclysm-DDA/pull/49501#discussion_r661132911